src/scene/materials/standard-material.js

```diff
@@ -1063,6 +979,46 @@ function _defineMaterialProps() {
     _defineAlias("glossVertexColor", "glossMapVertexColor");
     _defineAlias("opacityVertexColor", "opacityMapVertexColor");
     _defineAlias("lightVertexColor", "lightMapVertexColor");
+
+    // prefiltered cubemaps property
+    const getterFunc = function () {
```

Comment:
give these some less generic name please.

Reply:
these are just function-local variables? don't think they need bigger name.

Reply:
sure, those get minified anyways, but the name helps to read the code.

Reply:
Added a comment to each to make it clear :)